src/presentation/ui/component/databases/installedDatabaseTypeSummaryTable.templ

```diff
@@ -0,0 +1,56 @@
+package componentDatabases
+
+import (
+	"github.com/speedianet/os/src/domain/entity"
+	componentForm "github.com/speedianet/os/src/presentation/ui/component/form"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+	"strconv"
+)
+
+templ InstalledDatabaseTypeSummaryTable(
```

Comment:
I don't think this a component. Component is something that you reuse. This seems to be a part of a page. Components aren't page parts. I would move this back to the page. You can have multiple templ for each part of the page but the code must be on the page package and not on the component. Remember: component is fundamentally something reusable.

Reply:
Got it! I moved it back to the main page, and I'll pay attention to this next time.